imap/sievedir.c

```diff
@@ -166,7 +166,10 @@ EXPORTED struct buf *sievedir_get_script(const char *sievedir,
     buf_printf(&buf, "%s/%s", sievedir, script);
 
     int fd = open(buf_cstring(&buf), 0);
```

Comment:
Should we just buf_free(&buf) here since we do it whether open() succeeds or fails